git_cheatsheet.md

```diff
@@ -0,0 +1,104 @@
+# Git Cheatsheet
+### Modified from the [Github Education Material](https://education.github.com/git-cheat-sheet-education.pdf)
+### There are additional functions, but this is a basic introduction. 
+#### Note: Text in brackets must be replaced by object described within brackets: "[FIRSTNAME LASTNAME]" -> "Kevin Bacon" 
+## DEFINITIONS
+* **Local Repository** - A version of your repository that physically exists on your computer.
+* **Remote Repository** - A version of your repository that exists on a server (like GitHub's).
+* **Directory** - A folder on your computer. The working directory is the folder you are working in and can be changed using the `cd` command.
+* **Commit** - A snapshot of changes made to files. Files must be staged via the `git add` command. 
+* **Staged/Staging** - The process of identifying which files will be included in a commit. Can include all files in directory or specific files.
+
+## DIRECTORY NAVIGATION
+Change directory to explicit location:
+```
+cd PATH/TO/DESIRED/DIRECTORY/
+```
+Change directory to a folder "up" one level:
+```
+cd ..
+```
+*This is equivalent to `cd PATH/TO/DESIRED/`*
```

Comment:
A little unclear, as i'm not sure this specifies a base folder?

cd .. is this only if we actually got into the directory folder